### DISCUSSION TITLE
How to solve the stress strain curve of DP potential running lammps, which is basically in a straight line after each run? The trajectory of the dp potential atom is OK; It is also possible to use the eam momentum to run out.

### DISCUSSION THREAD

**Discussion by XuFanffei** [deepmodeling/deepmd-kit#2994] [Q&A]
### Summary

How to solve the stress strain curve of DP potential running lammps, which is basically in a straight line after each run? The trajectory of the dp potential atom is OK; It is also possible to use the eam momentum to run out.
This is the stresstrain diagram run from a script with the same eam momentum
![20231115-170258](https://github.com/deepmodeling/deepmd-kit/assets/122335090/1a800349-1367-4d64-ac19-01ea1cc9b7cc)
![20231115-170347](https://github.com/deepmodeling/deepmd-kit/assets/122335090/e488f376-8b1f-4665-ae69-152ab5c49d43)
Moreover, the stress should increase in the later stage, which seems to have remained unchanged, and it has indeed fluctuated in the early stage
It's quite strange to change the compute centroid/press/atom, which usually goes down and up
![20231115-170441](https://github.com/deepmodeling/deepmd-kit/assets/122335090/3d0561d5-6324-4725-a2f3-ea39812e7524)

### DeePMD-kit Version

2.1.5

### TensorFlow Version

2.8.0

### Details

null

**Comment by njzjz** [comment]
It seems related to https://github.com/deepmodeling/deepmd-kit/issues/1935
